### ISSUE TITLE
Rethinking Class-Balanced Methods for Long-Tailed Visual Recognition from a Domain Adaptation Perspective 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#32]
# TL;DR

In long tailed datasets, in which in the number of the data of each classes are very diverse, the distribution of test and training data is different for a small number of classes of data.
They considered this task as a domain-application problem, and used a meta-learning mechanism to train model parameters and per-data coefficients at the same time after training as usual.

# Why it matters:

# Paper URL
https://arxiv.org/abs/2003.10780

# Submission Dates(yyyy/mm/dd)
2020/03/24

# Authors and institutions
Muhammad Abdullah Jamal, Matthew Brown, Ming-Hsuan Yang, Liqiang Wang, Boqing Gong
- University of Central Florida 
- University of California at Merced 
- Google 

# Methods
They use a meta-learning mechanism to train model parameters and per-data coefficients at the same time after training as usual.
![image](https://user-images.githubusercontent.com/24524018/89614007-7c5bd380-d8be-11ea-9dba-45f25588b610.png)

# Results
Good results in long tailed datset
![image](https://user-images.githubusercontent.com/24524018/89613991-7108a800-d8be-11ea-983f-3a07206397ef.png)

# Comments
CVPR2020
